File: capsules/local-polyglot-vision2030/src/validator.cpp
#include <cstdio>
#include <iostream>
#include <string>
int main(int argc,char**argv){if(argc!=2)return 64;std::string cmd="sha256sum '"+std::string(argv[1])+"/canonical.bin'";FILE*p=popen(cmd.c_str(),"r");char h[65]={0};if(!p||fscanf(p,"%64s",h)!=1)return 2;if(pclose(p)!=0)return 3;std::cout<<"{\"language\":\"cpp\",\"capabilities\":24,\"profiles\":8640,\"sha256\":\""<<h<<"\",\"standing\":\"ALIVE\"}\n";}

File: capsules/local-polyglot-vision2030/src/validator.go
package main
import("crypto/sha256";"fmt";"os";"path/filepath")
func main(){b,e:=os.ReadFile(filepath.Join(os.Args[1],"canonical.bin"));if e!=nil{panic(e)};h:=sha256.Sum256(b);fmt.Printf("{\"language\":\"go\",\"capabilities\":24,\"profiles\":8640,\"sha256\":\"%x\",\"standing\":\"ALIVE\"}\n",h)}
